joss-paper/paper.md

```diff
@@ -0,0 +1,130 @@
+
+## Equivalent circuit fitting
+`impedance.py` equivalent circuit fitting combines an extremely flexible circuit definition scheme with a range of available circuit elements -- from simple L, R, C, and constant-phase elements to electrochemistry specific elements such as Warburg, Gerischer [@boukamp_gerischer_element_2003], and transmission-line models [@paasch_theory_1993]. Custom elements can be easily added to a single users installation or to the overall project by [creating an issue](https://github.com/ECSHackWeek/impedance.py/issues/new?assignees=&labels=&template=equivalent-circuit-element-request.md&title=%5BElement%5D). Additional model fitting features include parameter bounds, holding circuit elements constant, weighted fitting, and even saving/loading models to a human readable .json file. Fitting is performed by non-linear least squares regression of the circuit model to impedance data via `curve_fit` from the `scipy.optimize` package [@jones_scipy_2001].
+
+## Model selection
+One of the biggest challenges in extracting parameters from EIS spectra is quantitatively evaluating the fit of different models. `impedance.py` returns estimated parameter error bars with every fit and comes with a built in method for calculating the confidence bounds of a fit model.
+
+## Data visualization
+EIS is often qualitatively interpreted by examing the shape of a spectrum which relies on properly defined axes. `impedance.py` eliminates the tedium of manually formatting plots by providing built-in methods to generate clean, publication-ready figures including interactive plots via Altair [@vanderplas_altair_2018] and customizable Nyquist or Bode plots via matplotlib [@hunter_matplotlib_2007].
+
+
+# An example of the simple model API
+The documentation for `impedance.py` contains [a guide on getting started](https://impedancepy.readthedocs.io/en/latest/getting-started.html) and several examples of what a typical analysis workflow might look like using the package. Here we show how importing data, defining and fitting an equivalent circuit  model, and visualizing the results can be done with just a handful of lines in `impedance.py`:
+
+```python
+# 1. loading in data
+from impedance.preprocessing import readFile
+f, Z = readFile('exampleData.csv')
+
+# 2. importing and initializing a circuit:
+from impedance.models.circuits import CustomCircuit
+initial_guess = [1e-8, .01, .005, .1, .9, .005, .1, .9, .1, 200]
+circuit = CustomCircuit('L_0-R_0-p(R_1,CPE_1)-p(R_2,CPE_2)-Wo_1',
+                        initial_guess=initial_guess)
+
+# 3. fitting the circuit to the data
+circuit.fit(f, Z)
+print(circuit)
+```
+
+```text
+Circuit string: L0-R0-p(R1,E1)-p(R2,E2)-W1
```

Comment:
This circuit string is showing the old element naming format. I'm guessing this will be fixed if we re-run the example?